ui/core/src/config.localnet.json

```diff
@@ -8,8 +8,10 @@
   "nativeAsset": "rowan",
   "bridgebankContractAddress": "0x2C2B9C9a4a25e24B174f26114e8926a9f2128FE4",
   "keplrChainConfig": {
-    "chainId": "sifchain",
     "chainName": "Sifchain",
+    "chainId": "",
```

Comment:
should this be "sifchain" or was that causing problems?

Reply:
I changed it so only defined in one place. See parseConfig.ts

Reply:
I changed it so `chainId` only defined in one place. See parseConfig.ts